rdr_service/cron_default.yaml

```diff
@@ -249,5 +229,10 @@ cron:
   timezone: America/New_York
   schedule: every day 06:00
   target: offline
+- description: Report for AW3 Ready Samples Missing Data Files (Weekly)
```

Comment:
Can we prefix this with "Genomic"

Reply:
Done

Reply:
Sorry, can we add the Genomic to the url route as well?